tests/subtitleParser.spec.js

```diff
@@ -0,0 +1,159 @@
+import { waitFor } from '@testing-library/dom'
+import SubtitlesParser from '../src/SubtitlesParser'
+import { Log, initLightningSdkPlugin } from '../src/LightningSdkPlugins'
+import fs from 'fs'
+import { beforeAll, describe, it, jest, test } from '@jest/globals'
```

Comment:
These are injected automatically by Jest, I believe. The `jest/globals` part in `.eslintrc` should already suffice for ESLint not to throw errors. So I think you could remove this import

Reply:
yes, removed